solr/solr-ref-guide/modules/query-guide/pages/dense-vector-search.adoc

```diff
@@ -240,6 +240,66 @@ client.add(Arrays.asList(d1, d2));
 ====
 ======
 
+=== ScalarQuantizedDenseVectorField
+Because dense vectors can have a costly storage footprint, it may be worthwhile to use a technique called "quantization"
+to reduce the stored representation size at the cost of some precision.
+
+This dense vector type uses a conversion that projects a 32 bit float precision feature down to an 8 bit int (or smaller)
+by linearly mapping the float range down to evenly sized "buckets" of values that fit into an int. A more detailed explanation
+can be found in this https://www.elastic.co/search-labs/blog/scalar-quantization-101[blog post].
```

Comment:
I would encourage linking a resource that's not Elastic. I  don't think we want Elasticsearch reference in the Solr documentation, it would feel weird